lib/matplotlib/tests/test_backend_svg.py

```diff
@@ -145,3 +144,67 @@ def test_svgnone_with_data_coordinates():
         buf = fd.read().decode()
 
     assert expected in buf
+
+
+def test_gid():
+    """Test that object gid appears in output svg."""
+    from matplotlib.offsetbox import OffsetBox
+    from matplotlib.axis import Tick
+
+    fig = plt.figure()
+
+    ax1 = fig.add_subplot(131)
+    ax1.imshow([[1., 2.], [2., 3.]], aspect="auto")
+    ax1.scatter([1, 2, 3], [1, 2, 3], label="myscatter")
+    ax1.plot([2, 3, 1], label="myplot")
+    ax1.legend()
+    ax1a = ax1.twinx()
+    ax1a.bar([1, 2, 3], [1, 2, 3])
+
+    ax2 = fig.add_subplot(132, projection="polar")
+    ax2.plot([0, 1.5, 3], [1, 2, 3])
+
+    ax3 = fig.add_subplot(133, projection="3d")
+    ax3.plot([1, 2], [1, 2], [1, 2])
+
+    fig.canvas.draw()
+
+    gdic = {}
+    for idx, obj in enumerate(fig.findobj(include_self=True)):
+        if obj.get_visible():
+            gid = f"test123{obj.__class__.__name__}_{idx}"
+            gdic[gid] = obj
+            obj.set_gid(gid)
+
+    fd = BytesIO()
+    fig.savefig(fd, format='svg')
+    fd.seek(0)
+    buf = fd.read().decode()
+    fd.close()
+
+    def include(gid, obj):
+        # we need to exclude certain objects which will not appear in the svg
+        if isinstance(obj, OffsetBox):
+            return False
+        if isinstance(obj, plt.Text):
+            if obj.get_text() == "":
+                return False
+            elif not hasattr(obj, "axes") or obj.axes is None:
```

Comment:
The missing coverage is partial coverage on this line despite the next line being hit in all 5 tests.  The `axes` is a property so it should always exist on `Text` objects which means we always evaluate the second part of this if clause.  Investigating it locally...